src/components/BackButton/BackButton.js

```diff
@@ -1,10 +1,12 @@
+// @ts-nocheck
```

Comment:
Can you add a screenshot and description of the error on this place, please?

Reply:
![image](https://user-images.githubusercontent.com/61360270/142057570-cb87178e-db24-4d4f-8417-f011cc79c51b.png)
![image](https://user-images.githubusercontent.com/61360270/142057596-537a5917-e76d-432f-8700-18f46ce5f8ae.png)

Reply:
The one for `txt` is because `BackTxt` doesn't have props set properly.

The one for `id` should be ignored

@Katsu08 please lets add a ticket to modify that prop in `BackTxt`, is not the best way, I prefer the text to be in `children`. We can sync up about this offline